programs/x509/cert_write.c

```diff
@@ -230,6 +233,14 @@ int main( int argc, char *argv[] )
     mbedtls_ctr_drbg_context ctr_drbg;
     const char *pers = "crt example app";
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( mbedtls_platform_setup( &platform_ctx ) != 0 )
+    {
+        mbedtls_printf( "Failed initializing platform.\n" );
```

Comment:
please change to ` failed\n  !  mbedtls_platform_setup returned %d` for consistency